packages/terra-overlay/docs/README.md

```diff
@@ -1,7 +1,7 @@
 # Terra Overlay
 The Overlay component is a component that creates an semi-transparent overlay screen that blocks interactions with the elements underneath the display. There are two types of overlays: fullscreen and relative to its container.
 
-When creating as an overlay relative to its container, place the overlay component and the element(s) which should be blocked within the Overlay Container wrapper subcomponent.
+When creating as an overlay relative to its container, place the overlay component within the overaly wrapper and the element(s) which should be hidden by overlay within the content(children) wrapper of Overlay Container subcomponent.
```

Comment:
Typo: overlay

Reply:
Thank you.  [updated](https://github.com/cerner/terra-core/pull/2249/commits/4dd8524ffaaec55fcb3f9c13fedde57d4f555908)